src/Application/Messenger/Handler/StripePayoutHandler.php

```diff
@@ -119,7 +119,7 @@ public function __invoke(StripePayout $payout): void
             if ($donation->getDonationStatus() === DonationStatus::Collected) {
                 // We're confident to set donation status to paid because this
                 // method is called only when Stripe event `payout.paid` is received.
-                $donation->setDonationStatus(DonationStatus::Paid);
+                $donation->recordPayout($payoutId, $payoutInfo['created']);
```

Comment:
I had a closer look at some real payout data and I'm not sure this is the best value to use. It's sort of become less important now that may accounts get accelerated payouts but I don't think this is guaranteed.

`arrival_date` is sort of a date rather than a datetime (although it's still provided as a unix timestamp) but it might still be the better value to use. It seems like payouts which take 5 days have the later ETA as arrival date while those expected to land same day show the beginning of the day they were `created`.

Reply:
Thanks, will have a look at that in the morning. Being a date rather than a datetime reminds me I'm still interested in using https://github.com/brick/date-time especially as temporal progresses through tc39, but not just for this change with no real business logic and slighly less now I see that it's almost a year since brick/date-time has had a release.

Reply:
Thanks, switched to arrival date.